sktime/registry/_tags.py

```diff
@@ -225,12 +219,6 @@
         "bool",
         "can the classifier set n_jobs to use multiple threads?",
     ),
-    (
```

Comment:
this is perhaps still used in transformers that haven't been refactored yet, may cause errors

Reply:
might also be worth adding a todo here

Reply:
well iI can find no occurrences of coerce-X-to-numpy with find in files in pycharm. Are there any tests anywhere to see if tags declared exist in the registry?